Documents/start_contribution.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to JDI Light
+
+## Step 1: Clone the Repository 
+
+First, you'll need to clone the repository to your local machine. You can do this with the following command: git clone https://github.com/jdi-testing/jdi-light.git
```

Comment:
It is valid only for team members, also add a point for external - create a fork

Reply:
fixed